.github/CODEOWNERS

```
@@ -1,8 +1,7 @@
 # Lines starting with '#' are comments.
 # Each line is a file pattern followed by one or more owners.
 
-*                                    @MetaMask/devs
-
-/packages/permission-controller      @MetaMask/snaps-devs
-/packages/notification-controller    @MetaMask/snaps-devs
-/packages/rate-limit-controller      @MetaMask/snaps-devs
+*                                    @MetaMask/engineering
```

I'd recommend using this instead:

```suggestion
*                                    @MetaMask/extension-devs @MetaMask/mobile-devs  @MetaMask/shared-libraries 
```

Generally we avoid referencing the entire engineering group in codeowner rules to avoid pinging the entire organization for each PR review.

The closed equivalent to the purpose of the old devs team is: shared libraries, mobile platform, and extension platform. But we don't have teams yet for the client platforms (AFAIK), so the `[client]-devs` teams will have to do for now.

---

Ah, I didn't know we'd changed that policy. Should we update the module template?

---

Also I think there are two teams for Shared Libraries: `@MetaMask/shared-libraries` and `@MetaMask/shared-libraries-engineers`. Which one should we use?

---

Ah right; probably `shared-libraries-engineers` then

And yes, updating the module template would be great as well!

---

Okay, done here: [68654dd](https://github.com/MetaMask/core/pull/3924/commits/68654dda2ab897221eb81ec45185fd88d83eb594)

---

Hmm, feels like this would be a significant reduction in permissions for engineers working on controllers that aren't in one of the client teams (we have multiple engineers like this in snaps).